html-css-onboarding/movy-project/my-list.html

```diff
@@ -0,0 +1,427 @@
+        >
+          <svg
+            aria-hidden="true"
+            focusable="false"
+            data-prefix="fas"
+            data-icon="bars"
+            class="w-6"
+            role="img"
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 448 512"
+          >
+            <path
+              fill="currentColor"
+              d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
+            ></path>
+          </svg>
+        </button>
+        <div class="logo order-1">
+          <div class="oval"></div>
+          <a href="#" class="nav-item movy">Movy</a>
+        </div>
+        <div
+          class="collapse navbar-collapse flex-grow items-center order-3 lg:oder-2"
+          id="collapse-menu"
+        >
+          <ul
+            class="navbar-nav flex flex-col pl-0 list-style-none my-0 mr-auto pt-1.5 gap-4"
+          >
+            <li class="nav-item">
+              <a class="nav-link text-white" href="#">Home</a>
+            </li>
+            <li class="nav-item">
+              <a
+                class="nav-link text-white opacity-80 hover:opacity-90 focus:opacity-90 p-0"
+                href="#"
+                >Movies</a
+              >
+            </li>
+            <li class="nav-item">
+              <a
+                class="nav-link text-white opacity-80 hover:opacity-90 focus:opacity-90 p-0"
+                href="#"
+                >Series</a
+              >
+            </li>
+            <li class="nav-item">
+              <a
+                class="nav-link text-white opacity-80 hover:opacity-90 focus:opacity-90 p-0"
+                href="#"
+                >Recently Added</a
+              >
+            </li>
+            <li class="nav-item">
+              <a
+                class="nav-link text-white opacity-80 hover:opacity-90 focus:opacity-90 p-0"
+                href="#"
+                >My List</a
+              >
+            </li>
+          </ul>
+        </div>
+        <div class="profile-menu order-2 lg:order-3">
+          <img
+            id="search"
+            src="/html-css-onboarding/movy-project/assets/images/search.png"
+            alt=""
+          />
+          <img
+            id="profile"
+            src="/html-css-onboarding/movy-project/assets/images/profile-image.png"
+            alt=""
+          />
+          <img
+            id="arrow"
+            src="/html-css-onboarding/movy-project/assets/images/arrow-down-filled.png"
+            alt=""
+          />
+        </div>
+      </div>
+    </nav>
+    <div class="my-list-container">
+      <h2 class="font-medium leading-tight text-2xl content">My List</h2>
+      <div class="films">
```

Comment:
There's a little jump of the whole row when there's a hover over an item. Can we avoid this behavior so the non-hovered films only get displaced horizontally when we hover an item?